packages/three-vrm-springbone/src/VRMSpringBoneCollider.ts

```diff
@@ -10,9 +10,44 @@ export class VRMSpringBoneCollider extends THREE.Object3D {
    */
   public readonly shape: VRMSpringBoneColliderShape;
 
+  /**
+   * World space matrix for the collider shape used in collision calculations.
+   */
+  public readonly colliderMatrix = new THREE.Matrix4();
+
   public constructor(shape: VRMSpringBoneColliderShape) {
     super();
 
     this.shape = shape;
+    this.updateWorldMatrix(false, false);
```

Comment:
why is the initial calculation needed?

Reply:
Is not needed, removed it. The `VRMSpringBoneManager` already guarantees that `updateWorldMatrix` is called. Thought it would be nice to ensure the `colliderMatrix` was correct right after initialization, but in practice it's value is outdated as soon as the collider is added to the scenegraph and positioned.